spec/features/trainee_actions/filtering_trainees_spec.rb

```diff
@@ -192,6 +194,7 @@ def current_provider
   def given_all_trainees_are_from_a_single_source
     Trainee.with_apply_application.destroy_all
```

Comment:
We shouldn't be creating a load of trainees and then deleting them again in a spec. I realise this isn't introduced in this PR, just sayin'... Maybe we can look at this on fix-it-friday 😁